pkg/cmd/stack/deploy.go

```diff
@@ -274,18 +274,12 @@ func isSvcHealthy(ctx context.Context, svc *model.Service, stack *model.Stack, s
 	forwarder.Start(podName, stack.Namespace)
 	defer forwarder.Stop()
 	for _, port := range portsToTest {
-		url := fmt.Sprintf("http://%s:%d/", model.Localhost, port)
-		resp, err := http.Get(url)
+		url := fmt.Sprintf("%s:%d", model.Localhost, port)
+		_, err := net.Dial("tcp", url)
```

Comment:
we have similar code in the up sequence, which I think also does retries and other stuff. do you think we should use that here as well?

Reply:
It's already retrying but it will launch other services that are ready to be launch before retrying instead of waiting for a particular service to be available